bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/handler/SomfyTahomaActionGroupHandler.java

```diff
@@ -50,7 +50,7 @@ protected boolean isAlwaysOnline() {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
-        logger.debug("Action group: {} received command: {}", channelUID.getId(),command.toString());
+        logger.debug("Action group: {} received command: {}", channelUID.getId(), command.toString());
```

Comment:
I can remember we talked about this one somewhere else:

```suggestion
        logger.debug("Action group: {} received command: {}", channelUID.getId(), command);
```

Reply:
Yeah, you are right. Sorry